js_5/hm7Task2.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+// 1. Написать функцию, которая принимает на вход слово и проверяет, является ли это слово палиндромом
+
+const myWord = 'Топот'
+checkWordPalindrom(myWord)
+
+function checkWordPalindrom(word){
+    word = word.toLowerCase().split('')
+    let count = 0
+    for (let i = 0; i < word.length; i++) {
+        let startLetter = word[i]
+        let endLetter = word[(word.length - 1) - i]
+
+        if (startLetter === endLetter){
+            count += 1
+        }
+    }
+    if (count == word.length){
+        logWord(true, myWord)
+    } else {
+        logWord(false, myWord)
+    }
+}
```

Comment:
Хорошее решение! Давай немного добавим "красивостей")

```suggestion
function checkWordPalindrom(word){
    word = word.toLowerCase().split('')
    let count = 0
    for (let i = 0; i < word.length; i++) {
        if (word[i]!== word[(word.length - 1) - i]){
            return false
        }
    }
    return true
}
```

Reply:
А также в этой задаче есть еще 2 момента:
- если мы идем циклами - нужно ли нам идти по всей длине слова, или остановиться можно на середине?) мы ведь с двухсторон проверяем)
- можно решить через метод .reverse()